app-media-audio.html

```diff
@@ -45,7 +45,13 @@
             type: Object,
             readOnly: true,
             value: function() {
-              return new AudioContext();
+              if (window.AudioContext) {
+                // chrome requires class to be constructed directly
```

Comment:
What does it mean to be constructed directly?

Reply:
ah, I tried

```js
const audioContext = window.AudioContext || window.webkitAudioContext;
return new audioContext();
```

and it would fail on Chrome. Can't remember the error message rn

Reply:
Ah, yeah. SGTM.